includes/class-sensei-analysis.php

```diff
@@ -72,7 +72,7 @@ public function analysis_admin_menu() {
 	 */
 	public function enqueue_styles() {
 
-		wp_enqueue_style( 'woothemes-sensei-admin' );
+		wp_enqueue_style( 'sensei-admin' );
```

Comment:
As mentioned above, I don't see anywhere that this style is registered.

Reply:
ff867b018f67448905d9d6cdb4fc7a020aca57e8